modules/olm-building-operator-catalog-image.adoc

```diff
@@ -52,23 +51,19 @@ link:https://quay.io/[quay.io], tagging and pushing it to your mirror registry:
 +
 ----
 $ oc adm catalog build \
-    --appregistry-endpoint https://quay.io/cnr \
-    --appregistry-org redhat-operators \ <1>
-    --to=<registry_host_name>:<port>/<namespace>/redhat-operators:v1 <2>
+    --appregistry-org redhat-operators \//<1>
+    --to=<registry_host_name>:<port>/olm/redhat-operators:v1 \//<2>
+    [-a <path_to_registry_credentials>] \//<3>
+    [--insecure] <4>
 
 INFO[0013] loading Bundles                               dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605
-INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=manifests-829192605 load=bundles
-INFO[0013] directory                                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator load=bundles
-INFO[0013] found csv, loading bundle                     dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605 file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundles
-INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.package.yaml load=bundle
-INFO[0013] loading bundle file                           dir=/var/folders/st/9cskxqs53ll3wdn434vw4cd80000gn/T/300666084/manifests-829192605/3scale-operator file=3scale-operator.v0.3.0.clusterserviceversion.yaml load=bundle
 ...
-Uploading ... 244.9kB/s
-Pushed sha256:f73d42950021f9240389f99ddc5b0c7f1b533c054ba344654ff1edaf6bf827e3 to example_registry:5000/openshift/redhat-operators:v1
+Pushed sha256:f73d42950021f9240389f99ddc5b0c7f1b533c054ba344654ff1edaf6bf827e3 to example_registry:5000/olm/redhat-operators:v1
 ----
 <1> Organization (namespace) to pull from an App Registry instance.
-<2> Image repository tag to apply to the built catalog image. Specify a namespace
-and give it a tag, for example, `v1`.
+<2> Name your catalog image and include a tag, for example, `v1`.
+<3> If required, specify the location of your registry credentials file for `podman` or `docker`.
```

Comment:
Went with this phrasing because not sure what to suggest as the podman authfile example. The man page for podman-pull says:

```
     --authfile=path

       Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json, which is set using pod‐
       man  login.   If  the authorization state is not found there, $HOME/.docker/config.json is checked, which is
       set using docker login. (Not available for remote commands)
```

For my regular local user, that ends up being `/run/user/1000/containers/auth.json`. If `podman login` was run with sudo, it's `/run/containers/0/auth.json`, but then you'd also need to run the catalog commands with sudo to access it, AFAICT.

So should I suggest `${XDG_RUNTIME_DIR}/containers/auth.json` as what to use with `-a` when using podman?

Reply:
@Bowenislandsong WDYT? re: https://bugzilla.redhat.com/show_bug.cgi?id=1797026

Reply:
wording sgtm